### ISSUE TITLE
voting button not accessible

### ISSUE THREAD

**Issue by CarolingerSeilchenspringer** [liqd/a4-meinberlin#4137]
**URL:** https://meinberlin-dev.liqd.net/projekte/module/burgerinnenhaushalt-3-phasen-5/?mode=list#main
**user:** user without mouse
**expected behaviour:** I can vote
**behaviour:** vote button (both on list and detail page) has focus but hitting enter doesn't change vote. Also focus jumps from voting button to something invisible before going to next idea. 
**important screensize:**
**device & browser:** big sur/ Safari 15.1 /FF / chrome
**Comment/Question:** 

Screenshot?
